include/cuda/std/detail/libcxx/include/__utility/pair.h

```diff
@@ -309,6 +381,21 @@ struct _LIBCUDACXX_TEMPLATE_VIS pair
         return *this;
     }
 
+#if defined(__cuda_std__) && !defined(__CUDACC_RTC__)
+    __host__ _LIBCUDACXX_CONSTEXPR_AFTER_CXX17
+    pair& operator=(__conditional_t<
```

Comment:
```
C:\Users\Wes\Documents\workspace\libcudacxx\include\cuda\std\detail\libcxx\include\__utility/pair.h(386): error: invalid redeclaration of member function "cuda::std::__4::pair<_T1, _T2>::operator=(const cuda::std::__4::__conditional_t<<expression>, cuda::std::__4::pair<_T1, _T2>, cuda::std::__4::__nat> &) [with _T1=long, _T2=MoveOnly]"
(372): here
          detected during instantiation of class "cuda::std::__4::pair<_T1, _T2> [with _T1=long, _T2=MoveOnly]"
C:\Users\Wes\Documents\workspace\libcudacxx\.upstream-tests\test\std\utilities\tuple\tuple.tuple\tuple.assign\move_pair.pass.cpp(46): here
```

Reply:
Is this with windows?

Reply:
I have fixed that bug, by SFINAE-ing it away properly